src/variables.css

```diff
@@ -0,0 +1,31 @@
+:root {
+    --font-family-inter: 'Inter', sans-serif;
+
+    --font-weight-normal: 400;
+    --font-weight-medium: 600;
+    --font-weight-bold: 700;
+
+    --font-size-base: 1.4rem;
+    --font-size-title: 3rem;
+
+    --layout-mobile: 375px;
+    --layout-desktop: 1440px;
+
+    --color-green: hsl(75, 94%, 57%);
+    --color-white: hsl(0, 0%, 100%);
+    --color-gray700: hsl(0, 0%, 20%);
+    --color-gray800: hsl(0, 0%, 12%);
+    --color-gray900: hsl(0, 0%, 8%);
+
+    --border-radius: 2rem;
+
+    --margin-base: 3rem;
+}
+
+
+/* 
+
+### Font
+- Family: [Inter](https://fonts.google.com/specimen/Inter)
+
+*/
```

Comment:
Avoid comments if they are not documentation

Reply:
yes, sir! no problem, i will do as recommended :D